src/App.js

```diff
@@ -9,21 +10,26 @@ import Resource from './views/Resource';
 import Logout from './views/Logout';
 import Home from './views/Home';
 import './App.css';
+import logo from './img/iRODS-logo.png';
 
 class App extends Component {
   render() {
     return (
-      <div>
-        <Router>
-          <Home path="/home" />
-          <EditUser path='/user/edit' />
-          <User path="/user" />
-          <EditGroup path='group/edit' />
-          <Group path='/group' />
-          <Resource path='/resource' />
-          <Logout path='/logout' />
-          <Authenticate path="/" />
-        </Router>
+      <div className="app_body">
+        <div className="app_wrap">
+          <Router>
+            <Home path="/home" />
+            <EditUser path='/user/edit' />
+            <User path="/user" />
+            <EditGroup path='group/edit' />
+            <Group path='/group' />
+            <Resource path='/resource' />
+            <Logout path='/logout' />
+            <Authenticate path="/" />
+          </Router>
+        </div>
+        <hr />
+        <div className="app_footer"><div><img className="app_footer_logo" src={logo}></img>iRODS Consortium © 2021</div><div>Zone Management Tool Version: 0.1.0, {process.env.REACT_APP_GIT_SHA.substring(0,7)}</div></div>
```

Comment:
the version number (0.1.0) should come from a variable?

'iRODS Consortium' should be a link to https://irods.org